test/helpers/utils.go

```diff
@@ -301,3 +304,48 @@ func DNSDeployment() string {
 	}
 	return GetFilePath("provision/manifest/" + DNSEngine + "_deployment.yaml")
 }
+
+// getK8sSupportedConstraints returns the Kubernetes versions supported by
+// a specific Cilium version.
+func getK8sSupportedConstraints(ciliumVersion string) (go_version.Constraints, error) {
+	cst, err := go_version.NewVersion(ciliumVersion)
+	if err != nil {
+		return nil, err
+	}
+	// Make pre-releases part of the official release
+	strSegments := make([]string, len(cst.Segments()))
+	if cst.Prerelease() != "" {
+		for i, segment := range cst.Segments() {
+			strSegments[i] = strconv.Itoa(segment)
+		}
+		ciliumVersion = strings.Join(strSegments, ".")
+		cst, err = go_version.NewVersion(ciliumVersion)
+		if err != nil {
+			return nil, err
+		}
+	}
+	switch {
+	case CiliumV1_0.Check(cst):
+		return versioncheck.MustCompile(">= 1.8, <1.13"), nil
```

Comment:
This should be 1.7 for v1.0? The CI tests for v1.0 test against v1.7 in `ginkgo.Jenkinsfile`. Same for v1.1.

Reply:
bad copy-paste, fixed